packages/integration-tests/src/entities.ts

```diff
@@ -0,0 +1,77 @@
+export enum Buffer {
```

Comment:
Why is this file called entities? We should create a config file and constants file and move things there.

Reply:
renamed